### PR TITLE
goodbye, docs team

### PR DESCRIPTION
**Pull request by steveklabnik**

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Well, I guess I'll continue my PR on the std docs from the rustdoc team.

File: posts/inside-rust/2020-03-06-goodbye-docs-team.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+But documentation is a funny thing. It's really a cross-cutting concern. One
+team of folks writing docs for tons of other teams of folks doesn't really
+*work*, long-term. In the short term, it was an absolutely neccesary and good
+strategy. Today, it doesn't make as much sense. Let's look again at those original
+resources:
+
+* The standard library's documentation is pretty much filled out, and when new APIs
+  are added, the libs team writes some initial docs.
+* The book is maintinaed by Steve and Carol.
```
Comment (fwalch) [inline posts/inside-rust/2020-03-06-goodbye-docs-team.md]:
```suggestion
* The book is maintained by Steve and Carol.
```

**Comment by steveklabnik** [comment]
@GuillaumeGomez yep! we don't need a team to send in PRs.

**Comment by GuillaumeGomez** [comment]
No but it's that's the first team I ever joined. There is a lot of history in the docs team and it's definitely sad for me to see it getting closed. :'(

File: posts/inside-rust/2020-03-06-goodbye-docs-team.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+> as the team sees fit. Regular meetings will be important to coordinate
+> broader goals; and participation will be important for team members. We hold
+> meetings weekly.
+
+At the time, all of this was sorely needed. There weren't as many people working
+on Rust, and there wasn't that much documentation.
+
+But documentation is a funny thing. It's really a cross-cutting concern. One
+team of folks writing docs for tons of other teams of folks doesn't really
+*work*, long-term. In the short term, it was an absolutely neccesary and good
```
Comment (cuviper) [inline posts/inside-rust/2020-03-06-goodbye-docs-team.md]:
```suggestion
*work*, long-term. In the short term, it was an absolutely necessary and good
```

**Comment by nikomatsakis** [comment]
Are there changes e.g. to the website of teams report that are needed in light of this post?

**Comment by steveklabnik** [comment]
Yes, we would probably want to make some PRs there, but I don't know how to move an entire team to alumni, if we even do that? I don't think this change makes any member into an alumni; everyone who's still on the team is on other teams.

**Comment by XAMPPRocky** [comment]
@steveklabnik If you'd like I can make the PR and you can approve it?

**Comment by steveklabnik** [comment]
That would be great!

**Pull request merged**